internal/controllers/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -659,3 +659,11 @@ func (r *Reconciler) externalRemediationRequestExists(ctx context.Context, m *cl
 	}
 	return remediationReq != nil
 }
+
+func formatPercentSign(message string) string {
+	var out string
+	if strings.Contains(message, "%") {
+		out = strings.Replace(message, "%", "%%", -1)
+	}
+	return out
```

Comment:
This returns an empty string if the message does not contain "%"

Reply:
Couldn't this just be inlined as `strings.Replace(message, "%", "%%", -1)` ?

Reply:
Yeah, inlining would work too, I made it a function to make the issue clearer but I'll inline and update.

Reply:
Both fine for me. Just don't return an empty string if there's a message without "%" :)